doc/README.librexgen

```diff
@@ -2,6 +2,12 @@ Rexgen.txt - This document describes how to use the rexgen library, to perform
 regex expression work within JtR.  Rexgen library is copywrite Jan Starke,
 jan.starke@outofbed.org   The regex.c code in JtR by JimF, Spring 2014.
 
+WARNING
+
+Since maintenance for the rexgen library appears to have stopped for almost
+two years, support for --regex might have to be dropped in future releases!
```

Comment:
Maybe mention specific calendar years, as the "two years" reference would be non-specific to someone reading this years later.